src/Core/Models/Api/SendFileModel.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Bit.Core.Models.Data;
+using Bit.Core.Utilities;
+
+namespace Bit.Core.Models.Api
+{
+    public class SendFileModel
+    {
+        public SendFileModel() { }
+
+        public SendFileModel(SendFileData data, GlobalSettings globalSettings)
+        {
+            Id = data.Id;
+            Url = $"{globalSettings.Send.BaseUrl}/{data.Id}";
```

Comment:
Ah, I see where the URL is used now... although I still don't see a path that would match what's in the configuration JSON.